packages/core/http/core-http-router-server-internal/src/validator.ts

```diff
@@ -119,6 +120,8 @@ export class RouteValidator<P = {}, Q = {}, B = {}> {
   ): T {
     if (isConfigSchema(validationRule)) {
       return validationRule.validate(data, {}, namespace);
+    } else if (isZod(validationRule)) {
+      return validationRule.parse(data);
```

Comment:
Namespace is not passed here?

Reply:
Yeah, this is part of why Zod is not a drop-in replacement for Joi, no passing context down via the [parse](https://zod.dev/?id=parse) method.